g/gts/stone.yaml

```diff
@@ -0,0 +1,23 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : gts
+version     : 0.7.6
+release     : 1
+homepage    : https://gts.sourceforge.net/
+upstreams   :
+    - https://sourceforge.net/projects/gts/files/gts/0.7.6/gts-0.7.6.tar.gz : 059c3e13e3e3b796d775ec9f96abdce8f2b3b5144df8514eda0cc12e13e8b81e
+summary     : GNU Triangulated Surface Library
+description : |
+    GTS stands for the GNU Triangulated Surface Library
+license     : LGPL-2.0-or-later
+builddeps  :
+    - glib2-devel
```

Comment:
Could this be replaced with a pkgconfig builddep? I assume it wants `pkgconfig(glib-2.0)` since that's what it links against.